libraries/AC_Autorotation/AC_Autorotation.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
+
+#include <AP_Common/AP_Common.h>
+#include <AP_Param/AP_Param.h>
+#include <AP_Math/AP_Math.h>
+#include <AP_Logger/AP_Logger.h>
```

Comment:
I think the include of AP_Logger and AP_RPM can be moved to the .cpp file.  It's nice to reduce includes in the .h files because it reduces compile time.

Reply:
Good to know.  Have done as suggested.